sysutils/ntpstat/Portfile

```diff
@@ -37,3 +37,5 @@ destroot {
     xinstall -m 0755 -d ${destroot}${prefix}/share/doc/${name}
     xinstall -m 0644 ${worksrcpath}/COPYING ${destroot}${prefix}/share/doc/${name}
 }
+
+livecheck.version   11f1d49cf4041169e1f741f331f65645b67680d8
```

Comment:
I would just set `livecheck.type none` ; there have been no commits to this since 2013 so changes are very small that anything will happen upstream for this. Alternatively, just update to the latest commit and then it will all work; this is non-ideal, in between solution.

Reply:
I agree that it is unlikely the source will ever be updated.  The original source code was written by RedHat and RedHat does not sharing its source anymore!  :-)

I will update the PR and just use `livecheck.type none`